cfpb/cfgov-refresh
# Moving /students into cfgov-refresh

Moving /students into cfgov-refresh


## Changes

- Created Jinja template and modified css code to move `/students` into cfgov-refresh. I had to modify the header / footer css to use the new cf4 atomic styles. I left the existing styles in place until all the Wordpress pages could be converted.

## Testing

1. Visit `localhost:8000/students`.
2. Compare it to `/students` on consumerfinance.gov.
3. View source on ` localhost:8000/students`; Verify the meta tags and og tags are correct.
4. Visit `http://localhost:8000/students/knowbeforeyouowe/`.
5. Compare it to `/students` on consumerfinance.gov.
6. View source on ` http://localhost:8000/students/knowbeforeyouowe/`; Verify the meta tags and og tags are correct.
7. Visit `http://localhost:8000/students/helping-borrowers-find-ways-to-stay-afloat/`.
8. Compare it to `/students` on consumerfinance.gov.
9. View source on `http://localhost:8000/students/helping-borrowers-find-ways-to-stay-afloat/`; Verify the meta tags and og tags are correct.


## Screenshots


## Notes

- I didn't include include the `link` tags listed below:

```

<link rel='https://api.w.org/' href='https://www.consumerfinance.gov:443/wp-json/' />
<link rel="canonical" href="https://www.consumerfinance.gov:443/students/" />
<link rel='shortlink' href='https://www.consumerfinance.gov:443/?p=8331' />
<link rel="alternate" type="application/json+oembed" href="https://www.consumerfinance.gov:443/wp-json/oembed/1.0/embed?url=https%3A%2F%2Fwww.consumerfinance.gov%3A443%2Fstudents%2F" />

```
I can if folks deem it necessary.

## Todos

-

## Checklist
